### DISCUSSION TITLE
Generating Partial Schematics with `show` Command

### DISCUSSION THREAD

**Discussion by mmxsrup** [YosysHQ/yosys#4633] [Q&A] [answered]
I would like to generate a partial schematic that includes only a subset of cells from a very large module. When I use the `show` command in Yosys, it creates a full schematic, which is quite large. Is there a way to limit the schematic to only show certain cells within the module? Any advice on how to achieve this would be greatly appreciated.

**Comment by jix** [comment, marked as answer]
The `show` command (among many other commands) supports selections for this use case. The [manual chapter on selections](https://yosyshq.readthedocs.io/projects/yosys/en/latest/using_yosys/more_scripting/selections.html) has an introduction and some examples and the [command reference entry for the `select` command](https://yosyshq.readthedocs.io/projects/yosys/en/latest/cmd/select.html#cmd-select) is the reference for the selection syntax.

**Comment by mmxsrup** [comment]
The `select` command appears to be very feature-rich.
Thank you.
